Week-11/lightswitch/src/App.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react'
+import { useSelector , useDispatch} from 'react-redux';
+import {changeLight} from './action/action';
+import './App.css'
+
+const App = () => {
+  const myState = useSelector(state=>state.reducer);
+  const dispatch = useDispatch();
+  return (
+    <div className={myState ? 'dark' : 'lit'}>
+      <button onClick={()=>dispatch(changeLight(!myState))}>enable DarkMode</button>
```

Comment:
myState is an string here but you're treating it like a boolean value and adding not here in dispatch(changeLight(!myState)). This is where you should learn TypeScript.